ballet/update.py

```diff
@@ -0,0 +1,61 @@
+import git
+import os
+import tempfile
+import yaml
+
+from cookiecutter.main import cookiecutter
+
+from ballet.compat import pathlib
+from ballet.quickstart import generate_project
+
+YML_FILE = 'ballet.yml'
+
+def _find_ballet_dir(path):
+    '''
+    Searches for the directory containing the ballet path,
+    using the ballet.yml as a guide
+    '''
+    if os.path.exists(path / YML_FILE):
+        return path
+    for parent in path.parents:
+        parent_path = pathlib.Path(parent)
+        if os.path.exists( parent_path / YML_FILE):
+            return parent_path
+        else:
+            continue
+    return None
+
+def _create_replay(tempdir, name):
+    generate_project(replay=True, output_dir=tempdir)
+    return pathlib.Path(tempdir) / name
+
+def update_project():
+    path = _find_ballet_dir(pathlib.Path(os.getcwd()))
+    yml_file = open(path / YML_FILE)
```

Comment:
Be careful of isolated `open` calls, they may mean the rsource is not being closed.
```
with open(path / YML_FILE, 'r') as yml_file:
    ballet_yml = yaml.load(yml_file)
```

Reply:
Anyways, replace this with `ballet.project.find_configs`